### PR TITLE
move aggregations to baseComponents

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: ui/client/cypress/integration/components.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 describe("Components list", () => {
   const seed = "components"
 
-  const totalComponents = 93
+  const totalComponents = 90
```
Comment (arkadius) [inline ui/client/cypress/integration/components.ts:4]: do u have concept why it is reduced again? :)

**Review by arkadius — APPROVED** [review]

File: engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkBaseComponentProvider.scala [hunk 2]
```diff
@@ -4,17 +4,30 @@ import com.typesafe.config.Config
...
   override def resolveConfigForExecution(config: Config): Config = config
 
-  override def create(config: Config, dependencies: ProcessObjectDependencies): List[ComponentDefinition] = List(
-    ComponentDefinition("union", UnionTransformer),
-    ComponentDefinition("union-memo", UnionWithMemoTransformer),
-    ComponentDefinition("dead-end", SinkFactory.noParam(EmptySink))
-  )
+  override def create(config: Config, dependencies: ProcessObjectDependencies): List[ComponentDefinition] = {
+    List(
+      ComponentDefinition("union", UnionTransformer),
```
[the remark this answers is not in the document]
Comment (mproch) [reply engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkBaseComponentProvider.scala]: done

File: ui/client/cypress/integration/components.ts (hunk 1 above)
Comment (mproch) [reply ui/client/cypress/integration/components.ts:4]: Changes in DevProcessConfigCreator, I added base&kafka components to streaming-dev classpath in config

File: docs/Changelog.md [hunk 3]
```diff
@@ -23,7 +23,10 @@ Nussknacker versions
   Flink engine uses now fixed exception handler: FlinkExceptionHandler. All deprecated FlinkEspExceptionHandler implementations are removed.
 * [#2543](https://github.com/TouK/nussknacker/pull/2543) Eager parameters can have helpers injected.
 * [#2493](https://github.com/TouK/nussknacker/pull/2493/files) kafka configuration is now provided by components provider configuration, if not provided avroKryoGenericRecordSchemaIdSerialization default is set to true - previously false
-     
+* [#2569](https://github.com/TouK/nussknacker/pull/2569) Flink aggregations are now part of flinkBaseComponents. `flink-model-util` is no longer needed and is removed.
+           
+
+>>>>>>> 8a40acd38 (Cleanup)
```
Comment (dswiecki) [inline docs/Changelog.md:29]: remove

File: engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala [hunk 4]
```diff
@@ -17,6 +18,11 @@ class FlinkProcessVerifier(modelData: ModelData) extends StaticMethodRunner(mode
     val processId = processVersion.processName
     try {
       logger.info(s"Starting to verify $processId")
+      val printed = modelData.modelClassLoader.classLoader match {
+        case a: URLClassLoader => a.getURLs.toList
```
Comment (dswiecki) [inline engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala:22]: rename a -> cl

File: engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala [hunk 5]
```diff
@@ -17,6 +18,11 @@ class FlinkProcessVerifier(modelData: ModelData) extends StaticMethodRunner(mode
...
+        case a => a
+      }
+      logger.info("FLINKBASECOMPONENT classloader: " + printed)
```
Comment (dswiecki) [inline engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala:25]: Flink base component / flinkBaseComponent ?

File: engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala [hunk 6]
```diff
@@ -17,6 +18,11 @@ class FlinkProcessVerifier(modelData: ModelData) extends StaticMethodRunner(mode
     val processId = processVersion.processName
     try {
       logger.info(s"Starting to verify $processId")
+      val printed = modelData.modelClassLoader.classLoader match {
```
Comment (dswiecki) [inline engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessVerifier.scala:21]: classLoaderResources/classLoaderUrls ?

**Pull request merged**